src/main/java/frc/robot/Robot.java

```diff
@@ -108,16 +115,19 @@ public void robotInit() {
         SmartDashboard.putBoolean("Skip Reverse Auton Drive", false);
         SmartDashboard.putNumber("Angle Difference", 0.0);
         SmartDashboard.putBoolean("Using Lidar", true);
+        SmartDashboard.putNumber("Lidar distance", 0.0);
+        SmartDashboard.putNumber("Vision distance", 0.0);
         SmartDashboard.putBoolean("Force use lidar", false);
+        SmartDashboard.putBoolean("Two ball (true) / center search (false) Auton", true);
 
-        /*
-         * m_chooser.setDefaultOption("Auton1", auton1);
-         * m_chooser.addOption("Auton2", auton2);
-         * m_chooser.addOption("Auton3", auton3);
-         * m_chooser.addOption("Auton4", auton4);
-         * m_chooser.addOption("Auton5", auton5);
-         * SmartDashboard.putData(m_chooser);
-         */
+        autonTwoBall = new AutonTwoBall(swerve, shooter, ingestor);
+        autonCenterSearch = new AutonCenterSearch(swerve, shooter, ingestor);
```

Comment:
These are all Command subclasses. Why not make a single Command object that you initialize to the appropriate Auton command when grabbing the chosen auton?